bhr.js

```diff
@@ -605,13 +700,16 @@ window.onload = async function() {
       if (bug.hang) {
         bug.hang.duration += hang.duration;
         bug.hang.count += hang.count;
+        updateAnnotationStats(bug.hang.annotationStats, hang.annotations);
         continue;
       }
       hang.knownBug = bug;
       bug.hang = hang;
     }
     // This hang is not in our merged list yet, add it.
     hang.frames = undefined;
+    hang.annotationStats = {};
```

Comment:
I believe it would actually need to be something like `hang.ids.push([hang.id, hang.count])`, right?

Reply:
Possibly (I was initially thinking we would get the information back from gThread, but looking again at the code, that might not be practical). Maybe we would need to pass the entire hang object.

Reply:
FYI, I'm hoping in the future to change how we merge 'similar' stacks, so that the one we show on the right side by default is the one with the highest sort order (so longest total amount of time), and I would like to offer a way to see alternate stacks for the same hang. Doing this will require keeping more information (I was hoping it would only be the hang ids, but maybe it's the entire hang objects I'll need to keep to provide this view of alternative stacks).